packages/components/src/FilterBar/subcomponents/FilterBarMultiSelect/FilterBarMultiSelect.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { Key } from "react"
+import { Selection } from "@react-types/shared"
+import { FilterMultiSelect, getSelectedOptionLabels } from "@kaizen/select"
+import { RootProps } from "@kaizen/select/src/FilterMultiSelect/components/Root"
+import { useFilterBarContext } from "../../context/FilterBarContext"
+
+export type FilterBarMultiSelectProps = Omit<
+  RootProps,
+  "isOpen" | "setIsOpen" | "renderTrigger" | "label" | "selectedKey" | "trigger"
+> & {
+  id?: string
+}
+
+// This should technically be handled within the FilterMultiSelect
+// so that it only returns consumable formats instead of Set.
+// Someone fix please.
+export type ConsumableSelection = string | Key[] | undefined
```

Comment:
Given that `FilterBarDatePicker` and `FilterBarDRP` give `Date` and `DateRange` objects respectively, does it actually matter if we provide a `Set` to the consumer? They will probably need to do some conversion for the dates anyway to construct their query right? So asking them to convert the Set is probably fine too?